content/response-to-reviewers.md

```diff
@@ -163,32 +193,44 @@ Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/58>
 
-Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/60>
+We agree with the reviewer that, given PSB's location patterns, it should not be considered in this study.
+We have removed this symposium out of all of the analyses.
+In the revision, we focus on ISMB and RECOMB as the two society-associated meetings that are meant to represent the society globally, aim to have an international reach, and have been held in multiple continents.
+Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/62>.
 
 > \* The background in the study is created from the publications in well-selected journals (Bioinformatics, BMC Bioinformatics and PLOS Computational Biology). That background is appropriate and provides a useful characterization but is not necessarily the most appropriate. When one talks about a society, an alternative appropriate background would be the professional membership in that society (a society has responsibilities to recognize the work of its members well). The gender information in ISCB should be publicly available so the authors can check the trends and agreements with the current background. The authors could even be able to work out a solution with ISCB to apply the predictors to the data and more deeply test their hypotheses. The conclusions could hold or not and new patterns could emerge.
 
 Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/61>
 
-> \* Related to above, the selected journals for the background are of global reach (especially Bioinformatics because of rankings and $0 closed-access publication charges) whereas selected ISCB conferences (with their honorees) are more localized in Europe and North America for financial reasons. Considering the field as a whole, there are conferences that are limited to Asia, such as APBC or ISCB-Asia/Genome Informatics Workshop that are both of good quality and well known. APBC 2020 web site also shows that BMC Bioinformatics is among journals for their special issue. This suggests that the journals indeed better represent the field and that conferences are more geographically fragmented, including potentially ISCB conferences. This in turn means that the selection of conferences is critical for the outcome and appropriateness of this analysis. I think it would be appropriate to see a recognition of this fact in the manuscript and adjust the analysis.
-
-Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/62>
-
```

Comment:
This whole paragraph got moved up?

Reply:
Yes. These two points that the reviewer made are really one point, so I thought pointing them to one issue where we had the primary discussion is sufficient. I did refer #60 to #62.